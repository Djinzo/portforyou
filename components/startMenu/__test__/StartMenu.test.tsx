import StartMenus from "@components/startMenu/StartMenu";

import { render } from "@testing-library/react";

describe("StartMenu", () => {
  // TECH DEPT : give a nice comment
  it("should match snapshot", () => {
    const res = render(<StartMenus />).asFragment();
    expect(res).toMatchSnapshot();
  });
});
