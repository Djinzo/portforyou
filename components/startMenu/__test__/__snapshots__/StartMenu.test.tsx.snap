// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartMenu should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL gmISqY"
  >
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Programes
    </div>
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Documents
    </div>
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Settings
    </div>
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Find
    </div>
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Help
    </div>
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Run
    </div>
    <div
      class="sc-hKwDye eiYvEK"
    />
    <div
      class="sc-gsDKAQ cmyPxl"
    >
      Next mocked image Shutdown
    </div>
  </div>
</DocumentFragment>
`;
